src/pages/HomePageDeprecated.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useState, useEffect } from "react";
```

Comment:
Why are we keeping this around?

Reply:
Oh for some reason that wasn't highlighted, removed

Reply:
AD, keeping this around so that functionality doesn't break if you want to try that